### PR TITLE
Chore/release workflow, README updates

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

Adds the release workflow, updates README and others.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,69 +35,4 @@ Autonomous services that use Autonolas technology are implemented as [Finite Sta
...
-        "seconds_since_last_reset": 15.812911033630371,
-        "healthy": true,
-        "seconds_until_next_update": -5.812911033630371
-      }
-
-## Design limitations
-
-* It is recommended to always start with an empty score stream, as each time the service starts it will retrieve all mentions from Twitter and add the corresponding scores to those stored in the stream. If the service is ran for a second time using the same stream, and this stream already contains previous scores, users points would be doubled as with the current design the service has no memory of previous runs.
\ No newline at end of file
+To learn how to run the service, check the [IEKit tutorial](https://docs.autonolas.network/product/iekit/).
```
Comment (jmoreira-valory) [inline README.md:38]:
```suggestion
To learn how to run a demo service based on the IEKit, read the [IEKit techncical docs](https://docs.autonolas.network/product/iekit/).
```

File: README.md [hunk 2]
```diff
@@ -65,69 +35,4 @@ Autonomous services that use Autonolas technology are implemented as [Finite Sta
 
 ## Running the service
```
Comment (jmoreira-valory) [inline README.md:36]:
```suggestion
## Running a demo the service
```

Comment (jmoreira-valory) [reply README.md:36]: Place this section above, to be consistent with the sectioning order in https://docs.autonolas.network/product/iekit/

Comment (dvilelaf) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -2,40 +2,10 @@
...
-* CeramicWrite: the keeper writes the new scores to a Ceramic stream.
-* Verification: all the agents verify that the keeper has successfully written to the Ceramic stream by reading it and comparing it to the expected values.
-* WalletRead: verified wallets (addresses that have been linked to a Twitter handle) are read from a Ceramic stream.
-* NewTokens: newly minted tokens are retrieved and total scores are associated with their corresponding token.
-
-### Handler
-
-* An HttpHandler handles requests made to the service. Requests like ```curl localhost:8000/1```, where 1 here is the token id, will be responded with the token metadata.
-
-## Requirements
```
[the remark this answers is not in the document]
Comment (dvilelaf) [reply README.md:5]: I just left the requirements at the bottom to replicate the status of contribution service. For building the service the instructions already live on the docs.

File: README.md [hunk 4]
```diff
@@ -2,40 +2,16 @@
 An toolkit for autonomous services implementing a decentralized Impact Evaluator built with the [Open Autonomy](https://docs.autonolas.network/open-autonomy/) framework.
 
 The demo service for the IEKit tracks mentions of [@autonolas](https://twitter.com/autonolas) on Twitter, assigns scores to them and writes those scores to a [Ceramic](https://ceramic.network/) stream.
-The service then:
 
-1. reads verified wallets (a mapping between Twitter handles and Ethereum addresses) from another Ceramic stream,
-2. checks for users that have minted a token, and
-3. updates their token metadata according to their scores so NFT images are upgraded over time.
+## Running a demo the service
```
Comment (jmoreira-valory) [inline README.md:6]:
```suggestion
## Running a demo service
```

Comment (dvilelaf) [reply README.md]: There was a spelling issue so I included this change myself directly

**Review by jmoreira-valory — APPROVED** [review]

**Comment by dvilelaf** [comment]
I'm canceling the tests since this PR does not modify code.

**Review by Adamantios — APPROVED** [review]

**Pull request merged**